e2e/_suites/ingest-manager/ingest-manager_test.go

```diff
@@ -21,12 +21,22 @@ var stackVersion = "8.0.0-SNAPSHOT"
 // affecting the runtime dependencies (or profile)
 var profileEnv map[string]string
 
+// queryMaxAttempts is the number of attempts to query elasticsearch before aborting
+// It can be overriden by OP_QUERY_MAX_ATTEMPTS env var
+var queryMaxAttempts = 5
+
+// queryRetryTimeout is the number of seconds between elasticsearch retry queries.
+// It can be overriden by OP_RETRY_TIMEOUT env var
+var queryRetryTimeout = 3
+
```

Comment:
As a follow-up improvement, I'm considering converting this basic retry into the backoff strategy. So it's in the TODO